using-mobile-extensions/adobe-mobileservices/mobileservices-api-reference.md

```diff
@@ -0,0 +1,80 @@
+# Mobile Services Event Reference
+
+## Track Deep link  <a id="trackdeeplink"></a>
+
+You can use this API to track a deep link or a marketing link, provided the link contains a key `a.deeplink.id` and a corresponding non-null and user generated value. The link can either be a created on Adobe Mobile Services UI, or generated by another vendor.
```

Comment:
- In the 1st sentence, please replace "provided" to "as long as"

- In the 2nd sentence, please reword to:
"The link can be a created in the Adobe Mobile Services UI or be generated by another vendor."